### PR TITLE
{geo}[foss/2020b,intel/2020b] WRF v3.9.1.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 4 out of 5 (5 easyconfigs in total)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/17f17d68dcb73dbc1f0b8878b10a7b90 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@Micket No, there's something really wrong...
```
ranlib  libwrflib.a -ltirpc
ranlib: invalid option -- 'l'
ranlib: invalid option -- 'i'
ranlib: invalid option -- 'r'
ranlib: invalid option -- 'p'
ranlib: invalid option -- 'c'
```

**Comment by boegel** [comment]
> Test report by @Micket
> **FAILED**
> Build succeeded for 4 out of 5 (5 easyconfigs in total)
> vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
> See https://gist.github.com/17f17d68dcb73dbc1f0b8878b10a7b90 for a full test report.
> 
> ugh, probably some test suite or something and i is problematic on my build node

@Micket In your case, it's a different problem:

```
/bin/bash: /apps/Vera/software/MPI/intel/2020.4.304/impi/2019.9.304/WRF/3.9.1.1-dmpar/WRFV3/compile: /apps/Vera/software/Compiler/GCCcore/10.2.0/tcsh/6.22.02/bin/tcsh: bad interpreter: No such file or directory
```

Did something remove the `tcsh` installation while `WRF` was being installed?

**Comment by Micket** [comment]
It might have ran out of disk space, maybe. I was multitasking and didn't keep track.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3122.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8151dc19e244c8e6061bc59496e4ea44 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by akesandgren** [comment]
Please update to tcsh 6.02.23 from #12592 before merging

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    # WRF-3.9.1.1_netCDF-Fortran_separate_path.patch
+    '49ae0eb4c5bacd3b932be14ccb9ac2d28cbd70828b633d9f13860c92d92b1eb7',
+    # WRF-3.9.1.1_GCC-10-allow-argument-mismatch.patch
+    '3cc8a43838c7d821d02a7c64acbc08b730691af5ab6bc5f6ba0afd6356c4af01',
+    '9a8547d6c68b0f318f74063b690c5c8cfcd3437ebebe837dbd1d4a2dc1ad2427',  # WRF-3.9.1.1_libtirpc.patch
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('tcsh', '6.22.02'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb:37]: please upgrade to tcsh 6.22.03 from #12592

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    # WRF-3.9.1.1_netCDF-Fortran_separate_path.patch
+    '49ae0eb4c5bacd3b932be14ccb9ac2d28cbd70828b633d9f13860c92d92b1eb7',
+    # WRF-3.9.1.1_GCC-10-allow-argument-mismatch.patch
+    '3cc8a43838c7d821d02a7c64acbc08b730691af5ab6bc5f6ba0afd6356c4af01',
+    '9a8547d6c68b0f318f74063b690c5c8cfcd3437ebebe837dbd1d4a2dc1ad2427',  # WRF-3.9.1.1_libtirpc.patch
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('tcsh', '6.22.02'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb:37]: Same here, use newer tcsh from #12592

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    ('time', '1.9'),
+]
+
+dependencies = [
+    ('libtirpc', '1.2.6'),
+    ('JasPer', '2.0.14'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb:43]: 2.0.24 is the JasPer version for 10.2.0

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    ('time', '1.9'),
+]
+
+dependencies = [
+    ('libtirpc', '1.2.6'),
+    ('JasPer', '2.0.14'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb:43]: 2.0.24 is the JasPer version for 10.2.0

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('tcsh', '6.22.02'),
+    ('time', '1.9'),
+]
+
+dependencies = [
+    ('libtirpc', '1.2.6'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-foss-2020b-dmpar.eb:42]: Current version of tirpc for 10.2.0 is 1.3.1

File: easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('tcsh', '6.22.02'),
+    ('time', '1.9'),
+]
+
+dependencies = [
+    ('libtirpc', '1.2.6'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WRF/WRF-3.9.1.1-intel-2020b-dmpar.eb:42]: Current version of tirpc for 10.2.0 is 1.3.1

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
node3505.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/eff3eba063f94f94331108e712babf8d for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in total)
node3132.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/373cbaff9fe2584ebacd93afa8dba909 for a full test report.

**Pull request merged**